lib/libyear_bundler/models/gem.rb

```diff
@@ -39,7 +52,29 @@ def release_date(gem_name, gem_version, http)
           report_problem(gem_name, "Release date not found: #{gem_name}: #{e.inspect}")
         end
 
-        private
+        def release_date_from_other(gem_name, gem_version)
```

Comment:
This implementation is obviously based on https://github.com/jaredbeck/libyear-bundler/blob/v0.9.0/lib/libyear_bundler/models/gem.rb#L21-L41